wrapper_config.toml.template

```diff
@@ -9,3 +9,8 @@ maximum_heap_size = 512
 
 # Any further java run arguments (called before -jar).
 # java_args = ["-client", "-disablesystemassertions"]
+#
+# path of folder containing wurstscript.jar
```

Comment:
To be consistent with other comments, can you please capitalize `Path` and use prose sentence? So -

> Path of the folder containing the wurstscript jar.